packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.doc.tsx

```diff
@@ -10,25 +10,32 @@ const HoverCardBasicExampleCode = require('!raw-loader!office-ui-fabric-react/sr
 const HoverCardTargetExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.Target.Example.tsx') as string;
 const HoverCardPlainCardExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.PlainCard.Example.tsx') as string;
 
+const HoverCardBasicExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/HoverCard/HoverCard.Basic.Example.Codepen.txt') as string;
+const HoverCardTargetExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/HoverCard/HoverCard.Target.Example.Codepen.txt') as string;
+const HoverCardPlainCardExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/HoverCard/HoverCard.PlainCard.Example.Codepen.txt') as string;
+
 export const HoverCardPageProps: IDocPageProps = {
   title: 'HoverCard',
   componentName: 'HoverCard',
   componentUrl: 'https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/office-ui-fabric-react/src/components/HoverCard',
   componentStatus: HoverCardStatus,
   examples: [
     {
-      title: 'Example 1: Expanding HoverCard wrapping an element',
+      title: 'Expanding HoverCard wrapping an element',
```

Comment:
nit: I am ok with changing this example titles but could you also change the reference to `Example 1` and `Example 2` in [this file](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/HoverCard/docs/HoverCardOverview.md)

Reply:
Ah I hadn't noticed that the doc text references the example numbers. I'll revert this part of the change. I also decided to separate the HoverCard and GroupedList stuff (which was blocked waiting for official codeowner review) into a separate change because this one has been waiting long enough and it will probably be easier for the codeowners of those components to look at a smaller PR.